src/themes/teams/siteVariables.ts

```diff
@@ -6,19 +6,18 @@ import { pxToRem } from '../../lib'
 
 export const htmlFontSize = '14px' // what 1rem represents
 
-const blackRgbaFormat = (alpha: number): string => `rgba(37, 36, 36, ${alpha})`
-export const black = blackRgbaFormat(1)
+const blackRgbaFormat = (alpha: number): string => `rgba(37, 36, 35, ${alpha})`
+export const black = '#252423'
 export const gray01 = blackRgbaFormat(0.95)
```

Comment:
just out of curiosity - could you, please, suggest why this one remains to be the only which uses transparency instead of solid color?